woocommerce/woocommerce
# Bump actions in package release workflows to fix deprecated Node version

### Changes proposed in this Pull Request:

Both prepare-package-release.yml and package-release.yml are using old actions versions and throw a deprecated Node.js version warning.

Updated actions/checkout to v4 and peter-evans/create-pull-request to v7

### How to test the changes in this Pull Request:

Run the workflows. They should be successful and no deprecated Node.js warning should be logged.

See [test run](https://github.com/woocommerce/woocommerce/actions/runs/11958464406/job/33338032372) for Prepare package release
See [test run](https://github.com/woocommerce/woocommerce/actions/runs/11958811298) for Package release
